config/values/salaries.yml

```diff
@@ -1,5 +1,75 @@
+salaries_date_appliesfrom: 1 September 2023
 salaries:
+# Detailed salaries
+  leadingpractitioner:
+    englandmin: £47,417
+    englandmax: £72,085
+    londonfringemin: £48,723
+    londonfringemax: £73,393
+    outerlondonmin: £51,179
+    outerlondonmax: £75,850
+    innerlondonmin: £56,377
+    innerlondonmax: £81,051
+  headteacher:
+    minshortened: £53k
+    maxshortened: £139k
+    englandmin: £53,380
+    englandmax: £131,056
+    londonfringemin: £54,685
+    londonfringemax: £132,352
+    outerlondonmin: £57,124
+    outerlondonmax: £134,765
+    innerlondonmin: £62,304
+    innerlondonmax: £139,891
+## implemented above line
+# Summary salaries
+  starting:
+    min: £30,000
+    minshortened: £30k
+  fiveyears:
+    max: £41,333
+    maxrounded: £41,000
+    maxshortened: £41k
+  average:
+    classroomshortened: £40k
+  qualifiedteacher:
+    englandmin: £30,000
+    englandmax: £46,525
+    londonfringemin: £31,350
+    londonfringemax: £47,839
+    outerlondonmin: £34,514
+    outerlondonmax: £51,179
+    innerlondonmin: £36,745
+    innerlondonmax: £56,959
+  unqualifiedteacher:
+    englandmin: £20,598
+    englandmax: £32,134
+    londonfringemin: £21,933
+    londonfringemax: £33,464
+    outerlondonmin: £24,415
+    outerlondonmax: £35,954
+    innerlondonmin: £25,831
+    innerlondonmax: £37,362
+# Additional payments
+  tlr:
+    salarywithtlr: £43k
+    maximumrounded: £15,000
+    max: £15,690
+# Non-UK salary thresholds
+  skilledworkervisatraineeminfulltime:
+    innerlondon: £25,831
+    outerlondon: £24,415
+    londonfringe: £23,200
+    restofengland: £23,200
+  skilledworkervisatraineeminparttime:
+    restofengland: £23,200
+  skilledworkervisateacherminfulltime:
+    innerlondon: £36,745
+    outerlondon: £34,514
+    londonfringe: £31,350
+    restofengland: £30,000
+  skilledworkervisateacherminparttime: £23,200
+# Example variables - used in documentation
```

Comment:
@martyn-w Do we need rows 72 to 75 in here - I can't see why we'd need them for the content.md?

If not, I'll remove them as I think they are confusing

Reply:
Lines 72-75 are used in `docs/content.md` as an example YAML implementation:


```yaml
en:
  funding_widget:
    subjects:
      maths:
        name: "Maths"
        group: "Secondary"
        sub_head: "Maths - starting salary: %{salaries_example_min}"
        funding: "The opening date for applications is %{dates_example_opening} so get ready to apply soon."
```

Removing the example value(s) will not break the code but will stop the example from working if copied+pasted into a new file.

Reply:
OK - thanks - I'll leave for now (but perhaps clear up the config file slightly)